README.md

```diff
@@ -1,5 +1,22 @@
-This repository serves as the collection-point, ideation hub, and process behind the [InnerSource Commons'](http://innersourcecommons.org) InnerSource Patterns--a set of proven and reviewed solutions to InnerSource problems. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource methodologies](http://innersourcecommons.org/getting-started/). See below the list for more on what a pattern is, how to use them in your organization, and how to get involved.
+This repository contains the InnerSource Patterns collected by the [InnerSource Commons][isc-website]. These patterns illustrate beneficial activities and behaviors found in [organizations who apply InnerSource][isc-getting-started]. 
```

Comment:
(In general all changes look good - not sure if I am overly picky with this comment):

I always feel a lot depends on "first sentences" because they are a door opener.

Would you feel the following is a improvement or a regression?

```suggestion
This repository contains the InnerSource Patterns collected by the [InnerSource Commons][isc-website]. These patterns are InnerSource best practices codified in a specific format to make it easy to understand, evaluate, and reuse them. 
```

Reply:
Looks good to me, feel free to merge it in. However then you will also have to remove the link the reference `[isc-getting-started]` as you are not using that link anymore.